uat/testing-features/src/main/java/com/aws/greengrass/steps/MqttControlSteps.java

```diff
@@ -38,7 +38,7 @@ public class MqttControlSteps {
 
     private static final int DEFAULT_CONTROL_GRPC_PORT = 47_619;
     private static final int DEFAULT_MQTT_KEEP_ALIVE = 60;
-    private static final int DEFAULT_MQTT_CONNECT_TIMEOUT = 30;
+    private static final int DEFAULT_MQTT_TIMEOUT = 30;
```

Comment:
Please do not use that constant directly.
1. Create private variable and initialize it from DEFAULT_MQTT_TIMEOUT.
2. Use variable in any MQTT operation  which requires timeout.
3. Create building block with set that timeout
4. Do not use that building block in steps of 1st scenario.

Reply:
As has been described in https://klika-tech.atlassian.net/browse/GGMQ-68

1. step "Then connection for device "clientDeviceTest" is successfully established within 3 seconds" must be removed from the scenario ggmq-1.feature
2. building block "@Then("connection for device {string} is successfully established within {int} {word}")" must be removed from MqttControlSteps.java

Reply:
Fixed in a0d4b6547583e61e60859894ddacb5cb4b059e0d